app/src/main/java/com/eightbitlab/blurview_sample/MainActivity.java

```diff
@@ -94,4 +104,22 @@ public String getTitle() {
             return title;
         }
     }
+
+    @OnClick(R.id.enable)
+    void enable() {
+        if (!blurView.isEnabled())
```

Comment:
Let's keep it as simple as it can be. 
No need for additional checks, right? just ` blurView.enableBlur()` and that's it.

Same below